spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/AetherFactories.java

```diff
@@ -52,22 +47,12 @@ public static RepositorySystemSession newSession(RepositorySystem system, boolea
 		if (!workOffline) {
 			session.setUpdatePolicy(RepositoryPolicy.UPDATE_POLICY_ALWAYS);
 		}
+		session.setChecksumPolicy(RepositoryPolicy.CHECKSUM_POLICY_FAIL);
```

Comment:
I have no clue why the PR fails. Maybe it's because of this checksum policy? Can you comment this and let's rerun the build? (obviously I'm only guessing the reason)